docs/en/guides/nvidia-jetson.md

```diff
@@ -60,52 +67,48 @@ Execute the below command to pull the Docker container and run on Jetson. This i
 t=ultralytics/ultralytics:latest-jetson && sudo docker pull $t && sudo docker run -it --ipc=host --runtime=nvidia $t
 ```
 
-## Start without Docker
+After this is done, skip to [this section](#use-tensorrt-on-nvidia-jetson).
 
-### Install Ultralytics Package
+### Start without Docker
+
+#### Install Ultralytics Package
 
 Here we will install ultralyics package on the Jetson with optional dependencies so that we can export the PyTorch models to other different formats. We will mainly focus on [NVIDIA TensorRT exports](https://docs.ultralytics.com/integrations/tensorrt) because TensoRT will make sure we can get the maximum performance out of the Jetson devices.
 
 1. Update packages list, install pip and upgrade to latest
-
 ```sh
 sudo apt update
 sudo apt install python3-pip -y
 pip install -U pip
 ```
 
 2. Install `ultralytics` pip package with optional dependencies
-
 ```sh
 pip install ultralytics[export]
 ```
 
 3. Reboot the device
-
 ```sh
 sudo reboot
 ```
 
-### Install PyTorch and Torchvision
+#### Install PyTorch and Torchvision
 
 The above ultralytics installation will install Torch and Torchvision. However, these 2 packages installed via pip are not compatible to run on Jetson platform which is based on ARM64 architecture. Therefore, we need to manually install pre-built PyTorch pip wheel and compile/ install Torchvision from source.
 
 1. Uninstall currently installed PyTorch and Torchvision
-
```

Comment:
Code blocks require a line break, you've deleted these throughout this file which is not correct. 

If in doubt you can reformat Markdown files with PyCharm reformat, mdformat, or prettier.